app/screens/redeem-lnurl-withdrawal-screen/redeem-bitcoin-detail-screen.tsx

```diff
@@ -135,17 +140,28 @@ const RedeemBitcoinDetailScreen = ({
   navigation,
   route,
 }: StackScreenProps<RootStackParamList, "redeemBitcoinDetail">) => {
-  const { fiatSymbol } = useDisplayCurrency()
+  const { formatMoneyAmount, displayCurrency } = useDisplayCurrency()
 
   const { callback, domain, defaultDescription, k1, minWithdrawable, maxWithdrawable } =
     route.params.receiveDestination.validDestination
-  const minWithdrawableSatoshis = minWithdrawable / 1000
-  const maxWithdrawableSatoshis = maxWithdrawable / 1000
+  const minWithdrawableSatoshis: BtcPaymentAmount = {
+    amount: Math.round(minWithdrawable / 1000),
+    currency: WalletCurrency.Btc,
+  }
+  const maxWithdrawableSatoshis: BtcPaymentAmount = {
+    amount: Math.round(maxWithdrawable / 1000),
+    currency: WalletCurrency.Btc,
+  }
+  const canSetAmount = minWithdrawableSatoshis.amount !== maxWithdrawableSatoshis.amount
```

Comment:
```suggestion
  const amountIsDefined = minWithdrawableSatoshis.amount !== maxWithdrawableSatoshis.amount
```

maybe a better variable name?

Reply:
What about `amountIsFlexible`

Reply:
fine with that. I think that will be better than canSetAmount. the fact it can be interpreted as verb and not as a statement is where is it's confusing to me, because it's given state.